ecl/eclagent/eclgraph.cpp

```diff
@@ -1746,6 +1746,12 @@ void EclAgent::executeThorGraph(const char * graphName)
             {
                 throw deserializeException(reply);
             }
+            case DAMP_THOR_REPLY_COSTEXCEEDED:
+            {
+                Owned<IException> e = deserializeException(reply);
+                StringBuffer str;
+                throw new WorkflowException(e->errorCode(), e->errorMessage(str), 0, WorkflowException::ABORT, e->errorAudience());
```

Comment:
was this required (new reply code)? or could you have sent back the exception as a DAMP_THOR_REPLY_ERROR ?

Reply:
If I used DAMP_THOR_REPLY_ERROR, a generic error message is provided: 
![image](https://user-images.githubusercontent.com/11410590/76952999-dfe06700-6905-11ea-87c6-2421747fdad6.png)

Reply:
With DAMP_THOR_REPLY_COSTEXCEEDED, I can structure the message better:
![image](https://user-images.githubusercontent.com/11410590/76953139-20d87b80-6906-11ea-90ff-51f5c5f1fcec.png)
I was thinking about making this more generic.  However, I cannot think how else a generic version of this message would be used.

Reply:
what did it look like if you pass back the exception with a DAMP_THOR_REPLY_ERROR code?